pandas/core/indexing.py

```diff
@@ -1426,14 +1429,53 @@ class _LocIndexer(_LocationIndexer):
     - A list or array of labels, e.g. ``['a', 'b', 'c']``.
     - A slice object with labels, e.g. ``'a':'f'`` (note that contrary
       to usual python slices, **both** the start and the stop are included!).
-    - A boolean array.
+    - A boolean array, e.g. [True, False, True].
     - A ``callable`` function with one argument (the calling Series, DataFrame
       or Panel) and that returns valid output for indexing (one of the above)
 
     ``.loc`` will raise a ``KeyError`` when the items are not found.
 
     See more at :ref:`Selection by Label <indexing.label>`
 
+    See Also
+    --------
+    at : Selects a single value for a row/column label pair
+    iat : Selects a single value for a row/column pair by integer position
+    iloc : Selects group of rows and columns by integer position(s)
+
+    Examples
+    --------
+    >>> df = pd.DataFrame([[12, 2, 3], [0, 4, 1], [10, 20, 30]],
+    ...      index=['r0', 'r1', 'r2'], columns=['c0', 'c1', 'c2'])
+    >>> df
+        c0  c1  c2
+    r0  12   2   3
+    r1   0   4   1
+    r2  10  20  30
+    >>> df.loc['r1']
+    c0    0
+    c1    4
+    c2    1
+    Name: r1, dtype: int64
+    >>> df.loc[['r1', 'r2']]
+        c0  c1  c2
+    r1   0   4   1
+    r2  10  20  30
+    >>> df.loc['r0', 'c1']
+    2
+    >>> df.loc['r0':'r1', 'c0']
+    r0    12
+    r1     0
+    Name: c0, dtype: int64
+    >>> df.loc[[False, False, True]]
+        c0  c1  c2
+    r2  10  20  30
+    >>> df.loc[df['c1'] > 10]
+        c0  c1  c2
+    r2  10  20  30
+    >>> df.loc[df['c1'] > 10, ['c0', 'c2']]
+        c0  c2
+    r2  10  30
```

Comment:
Can you make a second example series or DataFrame where the index values are integers, but not `0-len(df)`? And then show how `.loc` uses the *labels* and not the positions?

In that second example, could you also show a slice like `df.loc[2:5]` and show that it's closed on the right, so the label 5 is included?